packages/bot/src/common/behaviors/dice/dice.behavior.base.ts

```diff
@@ -146,6 +153,40 @@ export abstract class DiceBehaviorBase extends BehaviorBase {
     return null;
   }
 
+
+
+  protected rollSpoileredV1(trigger: Trigger, logTag: string, content: string, context: BehaviorContext, requireDice: boolean): string[] | null {
+    const match = content.match(/([sS]#)(?:(\d+)#\s*)?(.*)/i);
```

Comment:
It looks like this might be able to handle both spoilered and unspoilered results with some minor tweaks.
I think that's beyond the scope of this PR tho.